feat: detect unsafe passwords (#862)
aiven/aiven-docs: docs/platform/howto/unsafe-passwords.md

---
title: Change unsafe passwords
---

> The Aiven Platform checks your email and password combination against a database of exposed credentials every time you log in and change your password.

Reviewer: Should this say 'every time you log in or change your password' instead of 'and'? Just checking if both actions trigger the credential check or only one of them.

Author: It's when you do one or the other thing, using 'and' would sound like you log in first then change your password.

> If Aiven detects an unsafe password, your login will be blocked until you

Reviewer: ```suggestion
If Aiven detects an unsafe password,  your login is blocked until you
```
